upgradability/forward-proxy/README.md

```diff
@@ -0,0 +1,8 @@
+# Smart contract upgradability using forward proxy pattern
+
+## Forward proxy
+Forward proxy is a contract that has a catch-all method which forwards all non-matching calls to the earlier specified logic contract.
```

Comment:
worth specifying which contracts' storage is used as the context for executing the said code (aka forwardcall vs delegatecall)